src/Resources/Resources/ChangeLog.md

```diff
@@ -21,6 +21,7 @@
 * Update references in .psd1 to use relative path
 * Fix an issue where template deployment fails to read a template parameter if its name conflicts with some built-in parameter name.
 * Updated policy cmdlets to use new api version 2019-09-01 that introduces grouping support within policy set definitions.
+* Update resource client to new version that retrieves providers and aliases at tenant level
```

Comment:
The only change was to retrieve policy aliases at tenant level right?  Not providers for Get-AzResourceProvider

Reply:
Correct. The alias cmdlet gets aliases from the providers call results. I'll reword this to avoid confusion on that point.